torch/quantization/fx/_equalize.py

```diff
@@ -211,13 +211,14 @@ def __new__(cls, input_activation=torch.nn.Identity, weight=torch.nn.Identity):
 
 def node_supports_equalization(node: Node, modules) -> bool:
     """ Checks if the current node supports equalization
-    Currently we only support nn.Linear and F.Linear layers
+    Currently we only support nn.Linear/F.Linear and nn.Conv/F.conv layers
     """
     if node.op == 'call_module':
         return (isinstance(modules[node.target], nn.Linear) or
-                isinstance(modules[node.target], nni.LinearReLU))
+                isinstance(modules[node.target], nni.LinearReLU) or
```

Comment:
nit: we can use type(modules[node.target]) in [nn.Linear, nni.LinearReLU, nn.Conv2d]